integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/InlinedWSDLEndpointTestCase.java

```diff
@@ -1,96 +1,49 @@
-/**
- * Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- * <p>
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We should not update the license header since this is not a new file or test case

Reply:
done